src/Context/WebApiContext.php

```diff
@@ -241,7 +264,8 @@ public function theResponseShouldNotContain($text)
     public function theResponseShouldContainJson(PyStringNode $jsonString)
     {
         $etalon = json_decode($this->replacePlaceHolder($jsonString->getRaw()), true);
-        $actual = $this->response->json();
+        $actual = version_compare(ClientInterface::VERSION, '6.0', '>=')
```

Comment:
Why not using ``$this->response instance of ResponseInterface`` instead, being consistent with the way the check is done in ``printResponse``.